app/src/features/apiClient/contexts/apiClient.tsx

```diff
@@ -160,23 +166,23 @@ export const ApiClientProvider: React.FC<ApiClientProviderProps> = ({ children }
       });
   }, [uid, teamId]);
 
+  useEffect(() => {
+    trackUserProperties(apiClientRecords);
+  }, [apiClientRecords]);
+
```

Comment:
We need to make sure that this doesn't fire consequetively. If it does then can we add debounce here?